docs/content/documentation/templates/overview.md

```diff
@@ -186,7 +187,28 @@ template:
         ["2", "Printing"]
     ],
 }
- ```
+```
+
+#### Remote content
+
+Instead of using a file, you can load data from a remote URL. This can be done by specifying a `url` parameter to `load_data` rather than `file`.
+
+```jinja2
+{% set response = load_data(url="https://api.github.com/repos/getzola/zola") %}
+{{ response }}
+```
+
+By default, the response body will be returned with no parsing. This can be changed by using the `format` argument as above.
```

Comment:
Hmm I guess we need `plain` because of that? How about making the `format` argument mandatory?

Reply:
I could make it mandatory, but the fact it bases it off the file extension is a really nice touch for local data files. Having to specify that my `data.json` file is a JSON file feels unnecessary when it's so easy to guess from the file extension, and change it by default if required.

Reply:
I'm not sure myself what is the best choice there. I kind of like the nice UX for ending in JSON but in practice for remote data you will need to specify the format so it feels a bit like it could surprise the end user when something works for local data but not remote.

I would tend towards removing the special casing but not 100% convinced.

Reply:
I think the fact it will throw errors at build time complaining for a `kind` argument should eliviate this. Although perhaps a very descriptive error message / docs piece about it would be better?